README.md

```diff
@@ -85,3 +85,27 @@ To access the web-page of the application, open your browser and go to:
 ```
 http://localhost
 ```
+
+If you're using Linux and you're having trouble getting the above to work, then run:
+```bash
+minikube service istio-ingressgateway -n istio-system
+```
+This will show a number of different ports for the Istio `Ingress Gateway`. The output should look something like tihs:
+```
+|--------------|----------------------|-------------------|---------------------------|
+|  NAMESPACE   |         NAME         |    TARGET PORT    |            URL            |
+|--------------|----------------------|-------------------|---------------------------|
+| istio-system | istio-ingressgateway | status-port/15021 | http://192.168.49.2:30693 |
+|              |                      | http2/80          | http://192.168.49.2:31005 |
+|              |                      | https/443         | http://192.168.49.2:31659 |
+|--------------|----------------------|-------------------|---------------------------|
+[istio-system istio-ingressgateway status-port/15021
```

Comment:
The output looks a little different on my machine:

![image](https://github.com/remla23-team06/operation/assets/12817322/a443b538-9e20-4e1a-95c1-b1629a36b239)

```
|--------------|----------------------|-------------|------------------------|
|  NAMESPACE   |         NAME         | TARGET PORT |          URL           |
|--------------|----------------------|-------------|------------------------|
| istio-system | istio-ingressgateway |             | http://127.0.0.1:46387 |
|              |                      |             | http://127.0.0.1:46353 |
|              |                      |             | http://127.0.0.1:44943 |
|--------------|----------------------|-------------|------------------------|
[istio-system istio-ingressgateway  http://127.0.0.1:46387
http://127.0.0.1:46353
http://127.0.0.1:44943]
```